### ISSUE TITLE
out_norikra doesn't retry to prepare any targets after the thread failed

### ISSUE THREAD

**Issue by yoheimuta** [norikra/fluent-plugin-norikra#7]
Sometimes (a few times a day in our system) I encountered the failure to prepare a target with error messages like below.

```ruby
2015-04-01 23:19:47 +0900 [error]: norikra/output.rb:131:rescue in prepare_target: failed to prepare target:stdout_ad norikra="norikra1.mtburn.internal:26571" error=Errno::ETIMEDOUT message="Connection timed out - connect(2) for \"norikra1.mtburn.internal\" port 26571 (http://norikra1.mtburn.internal:26571)"
2015-04-01 23:19:47 +0900 [error]: norikra/output.rb:99:block in register_worker: Failed to prepare norikra data for target:stdout_ad
```

I could simulated this situation that [the thread to run register_worker](https://github.com/norikra/fluent-plugin-norikra/blob/master/lib/fluent/plugin/norikra/output.rb#L51) stopped after the same failure.

- Because [@norikra_started.push(t)](https://github.com/norikra/fluent-plugin-norikra/blob/master/lib/fluent/plugin/norikra/output.rb#L100) throw [NoMethodError](http://docs.ruby-lang.org/ja/2.2.0/class/NoMethodError.html).

This line has remained from [v0.0.1](https://github.com/norikra/fluent-plugin-norikra/commit/e93cb95b06748b2d545f02b169a441d0a22fd9ab#diff-8aaa29178da03cdf26e7d019efeccd0bR207), so I supposed this exception means something important.

But I want to avoid this failure (and subsequent forever [RuntimeError](https://github.com/norikra/fluent-plugin-norikra/issues/4) until td-agent restart).

Is there any idea to avoid this failure ?

https://github.com/norikra/fluent-plugin-norikra/issues/6 is a similar issue about result except for type.

**Comment by tagomoris** [comment]
Oh, this is bogus bug...

**Comment by yoheimuta** [comment]
Thank you for reply. I pulled request https://github.com/norikra/fluent-plugin-norikra/pull/8.

**Comment by tagomoris** [comment]
I fixed this bug at a809183 and released `v0.3.0`. Targets should be retried.
Thank you for reporting!

**Comment by yoheimuta** [comment]
Thanks !
